runtime/autoload/dist/ft.vim

```diff
@@ -1218,5 +1218,13 @@ export def FTvba()
   endif
 enddef
 
+export def IsSafeExecutable(filetype: string, executable: string): bool
+  var cwd = getcwd()
+  return get(g:, filetype..'_exec', get(g:, 'plugin_exec', 0))
```

Comment:
Does this line actually work without whitespace around the `..` string concat operator?

Reply:
👀 probably not, but I haven't actually tested it. I'll add the whitespace.